src/components/Select/Select.scss

```diff
@@ -155,9 +155,11 @@ $stacking-order: (
     right: 0;
     bottom: 0;
     left: 0;
-    border: border-width() solid var(--p-border);
+    border: border-width() solid var(--p-border-neutral-subdued);
+    border-bottom-color: var(--p-border-shadow-subdued);
     border-radius: var(--p-border-radius-base);
-    background-color: var(--p-surface);
+    background: var(--p-surface);
```

Comment:
Should we keep it as `background-color`?

Reply:
I was wondering this as well and was going to ask you about it. I'll double check to make sure this wasn't intentional

Reply:
Thanks Kyle ✨